python/cuml/linear_model/linear_regression.pyx

```diff
@@ -171,7 +172,8 @@ class LinearRegression:
            Dense vector (floats or doubles) of shape (n_samples, 1)
 
         """
-
+        print("self.algo : ",self.algo)
```

Comment:
I don't think we should print this always, perhaps as part of a verbose function argument?